src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+/**
+ * @author dogmax296
+ * @link https://github.com/dogmax296
+ */
```

Comment:
I don't think we need this, lets remove this dock
(same for other classes)

Reply:
done